README.md

```diff
@@ -100,43 +149,45 @@ function App() {
 ReactDOM.render(<App />, document.querySelector('#app'));
```

Comment:
This is outdated for React 18. I have updated the codesandbox linked in https://mui.com/material-ui/getting-started/usage/#quick-start